### ISSUE TITLE
precision (maximum length) of a string column in ResultSetMetaData is always zero

### ISSUE THREAD

**Issue by jpoernomoatsafe** [olavloite/spanner-jdbc#65]
Hi, 

What would be the recommended way to retrieve the maximum length of a `string` column?

Currently, to get a table's metadata, I do the following:

```
Statement statement = jdbcConnection.createStatement();
ResultSet resultSet = statement.executeQuery("select * from myTable where 1=0");
ResultSetMetaData metadata = resultSet.getMetaData();
int i = INDEX_OF_STRING_COLUMN;
metadata.getPrecision(i); // always returns 0
metadata.getColumnDisplaySize(i); // always returns 50
```
the problem is, both `metadata.getPrecision(i)` and `metadata.getColumnDisplaySize(i)` are not returning the correct maximum length of the string column. 

I've tried string columns with max lengths of `100` and `MAX`. I've also tried this when the string column held over 50 characters. 

The docs for `ResultSetMetaData` seem to suggest at least one of these two methods should return the maximum length of the string column.
https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSetMetaData.html#getPrecision(int)
https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSetMetaData.html#getColumnDisplaySize(int)

**Comment by olavloite** [comment]
This problem was probably already fixed in an earlier version for queries that did not use quoted table names. As of version 1.0.8 it will also work for queries where the table names have been quoted.

**Comment by olavloite** [comment]
Released 1.0.8
